lua/lspconfig/emmet_ls.lua

```diff
@@ -0,0 +1,27 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+local server_name = 'emmet_ls'
+
+configs[server_name] = {
+  default_config = {
+    cmd = { 'emmet-ls', '--stdio' },
+    filetypes = { 'html', 'css' },
```

Comment:
Not sure if these filetypes should be expanded? What about XML and JSX `typescriptreact`, `javascriptreact`? @aca any thoughts?

Reply:
I think I need to check whether it is working fine on react project. 

By the way, thanks for opening the issue.